redev_types.h

```diff
@@ -24,6 +24,6 @@ using CVs = std::vector<CV>;
 
 enum class ProcessType { Client = 0, Server = 1 };
 enum class TransportType { BP4 = 0, SST = 1 };
-
+enum class CommType{ Ptn = 0, Global = 1 };
```

Comment:
Let's change this to `Partitoned` instead of `Ptn`. It's possible @cwsmith will disagree with me here, but I think it will be more clear to have the full name spelled out what the distinction is between the two options.

Counter argument is we use `Ptn` for the partition class.